README.md

```diff
@@ -3,3 +3,17 @@
 # `lnschema_core`
 
 `lnschema_core` is documented & tested within `lamindb`: [github.com/laminlabs/lamindb](https://github.com/laminlabs/lamindb).
+
+## Developers' Note
+
+This package implements models but leaves concrete implementations of non-ORM methods (such as `Artifact.cache`) empty. The methods are implemented in `lamindb` and then the model classes are monkey-patched there as well.
```

Comment:
> This package implements models but leaves concrete implementations of non-ORM methods (such as `Artifact.cache`) empty.

Can we say:

> This package defines ORMs ...

Then it doesn't say "model" in one part and "ORM" in the other. In general I think "ORM" is superior because "lamindb.Record" is an ORM, and "django.db.models.Model" is an ORM and the SQLAlchemy etc. variants are also ORMs etc.

Re:
> The methods are implemented in `lamindb` and then the model classes are monkey-patched there as well.

I'd rather do
> The methods are implemented in `lamindb` and the implementations are added to the ORM classes dynamically ("monkey-patching").